support/metrics/metricsTracker.go

```diff
@@ -75,6 +76,11 @@ type commonProps struct {
 	EnabledFeatureMonitoring         bool    `json:"enabled_feature_monitoring"`
 	EnabledFeatureFilters            bool    `json:"enabled_feature_filters"`
 	EnabledFeaturePostgres           bool    `json:"enabled_feature_postgres"`
+	OperationalBuffer                float64 `json:"operational_buffer"`
+	OperationalBufferNonNativePct    float64 `json:"operational_buffer_non_native_pct"`
+	SimMode                          bool    `json:"sim_mode"`
+	LogPrefix                        string  `json:"log_prefix"`
```

Comment:
`logPrefix` should be a `bool` -- we want to only know if they have enabled the feature.
in fact, can we put this above operationalBuffer and call this `EnabledFeatureLogging`?